css-two-column-layout/styles.css

```diff
@@ -0,0 +1,83 @@
+body {
+  max-width: 800px;
+  font-family: 'Karla', sans-serif;
+  font-style: normal;
+}
+
+#container {
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.col-60 {
+  width: 60%;
+}
+
+.col-half {
+  width: 50%;
+  padding: 25px 40px;
+}
+
+.col-three-fourths {
+  width: 75%;
+}
+
+h1 {
+  font-weight: 600;
+  letter-spacing: 2px;
+  margin-bottom: 0;
+  text-align: center;
+}
+
+p {
+  color: rgb(105, 103, 103);
+  font-size: 11px;
+  line-height: 20px;
+}
+
+p.header {
+  text-align: center;
+}
+
+img {
+  width: 90px;
+  height: 90px;
+  float: left;
```

Comment:
Instead of using `float` here, I would set `.group` to be `display: flex`, which should allow your elements to sit on the same line without the need to use `float`. In general, `float` is an archaic tool that should be avoided, unless you specifically want to wrap text around an image, as in a newspaper layout.